src/pykeen/training/slcwa.py

```diff
@@ -43,10 +44,14 @@ def __init__(
         :param negative_sampler_cls: The class of the negative sampler
         :param negative_sampler_kwargs: Keyword arguments to pass to the negative sampler class on instantiation
          for every positive one
+        :param automatic_memory_optimization: bool
+            Whether to automatically optimize the sub-batch size during
+            training and batch size during evaluation with regards to the hardware at hand.
         """
         super().__init__(
             model=model,
             optimizer=optimizer,
+            automatic_memory_optimization=automatic_memory_optimization,
```

Comment:
what about lcwa.py?

Reply:
Not nevessary since lcwa's training loop doesn't override the base class's `__init__()`